app/views/answers/_answer.html.erb

```diff
@@ -0,0 +1,36 @@
+<div id="answer-<%= answer.id %>" class='answer'>
+  <div class="voting-data">
+    <% if answer.voted.include?(current_user.id)%>
+      <%= render partial: 'answers/voted', locals: {answer: answer}%>
+    <% else %>
+     <%= render partial: 'answers/unvoted', locals: {answer: answer}%>
+    <% end %>
+  </div>
+
+  <% if answer.id == question.best_answer_id %>
```

Comment:
a best_for? on the Answer model might make this easier to read. You could do
`if answer.best_for?(question)`